libkpod/container_data.go

```diff
@@ -253,7 +252,7 @@ func getOCIContainer(store storage.Store, container string) (*oci.Container, err
 	return oci.NewContainer(ctr.ID, name, containerPath, m.Annotations[annotations.LogPath], nil, labels, kubeAnnotations, img, &metadata, ctr.ImageID, tty, stdin, stdinOnce, false, false, containerDir, created, m.Annotations["org.opencontainers.image.stopSignal"])
 }
 
-func getOCIRuntime(store storage.Store, container string) (*oci.Runtime, error) {
+func (c *ContainerServer) getOCIRuntime(container string) (*oci.Runtime, error) {
```

Comment:
I think we should work based on the in-memory state. It shouldn't be a significant issue for kpod once we get the PR in to sync on-disk state when initializing, as we only really do one operation and then exit. Obviously the container could be changed on disk between initialization and calling inspect, but that won't really be fixed until we get proper on-disk state management merged.